OpenIPSL/Electrical/Controls/PSAT/AVR/AVRTypeII.mo

```diff
@@ -1,26 +1,26 @@
 within OpenIPSL.Electrical.Controls.PSAT.AVR;
 model AVRTypeII "PSAT AVR Type 2"
 
-  Modelica.Blocks.Interfaces.RealInput v "Generator termminal voltage (pu)"
+  Modelica.Blocks.Interfaces.RealInput v "Generator termminal voltage [pu]"
     annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
         iconTransformation(extent={{-140,-80},{-100,-40}})));
-  Modelica.Blocks.Interfaces.RealOutput vf "Filed voltage (pu)" annotation (
+  Modelica.Blocks.Interfaces.RealOutput vf "Filed voltage [pu]" annotation (
       Placement(transformation(
         extent={{-10,-10},{10,10}},
         origin={110,0}), iconTransformation(extent={{100,-20},{140,20}})));
   Modelica.Blocks.Interfaces.RealInput vref
-    "Reference generator terminal voltage (pu)" annotation (Placement(
+    "Reference generator terminal voltage [pu]" annotation (Placement(
         transformation(extent={{-140,40},{-100,80}}), iconTransformation(extent
           ={{-140,40},{-100,80}})));
-  parameter Modelica.SIunits.PerUnit vrmin=-5 "Minimum regulator voltage (pu)";
-  parameter Modelica.SIunits.PerUnit vrmax=5 "Maximum regulator voltage (pu)";
-  parameter Real Ka=100 "Amplifier gain (pu/pu)";
-  parameter Modelica.SIunits.Time Ta=0.5 "Amplifier time constant (s)";
-  parameter Real Kf=0.15 "Stabilizer gain (pu/pu)";
-  parameter Modelica.SIunits.Time Tf=0.1 "Stabilizer time constant (s)";
-  parameter Real Ke=0 "Field circuit integral deviation (pu/pu)";
-  parameter Modelica.SIunits.Time Te=0.2 "Field circuit time constant (s)";
-  parameter Modelica.SIunits.Time Tr=0.001 "Measurement time constant (s)";
+  parameter Modelica.SIunits.PerUnit vrmin=-5 "Minimum regulator voltage";
+  parameter Modelica.SIunits.PerUnit vrmax=5 "Maximum regulator voltage";
```

Comment:
Why Modelica.SIunits.PerUnit and not SI.PerUnit?

Reply:
It was because I left as it was implemented before...

Reply:
Now it is fixed once again